### PR TITLE
feat: add KitchenStories scraper

### LINKED ISSUE #547: Request kitchenstories.com
We always enjoy adding support for new websites to the library!

To help us out, please check that recipes published on the website you're requesting are public (we can't currently scrape recipes that require an account login) and add sample recipe URL(s) below:

- https://www.kitchenstories.com/en/recipes/spaghetti-aglio-e-olio-with-broccoli
- https://www.kitchenstories.com/de/rezepte/pasta-aglio-e-olio-mit-brokkoli (the same in german)
- https://www.kitchenstories.com/en/recipes/tiktok-s-vegan-green-goddess-salad
- https://www.kitchenstories.com/en/recipes/cauliflower-steak

Can you write Python and would you like to help add the scraper yourself?  We'd be glad for your assistance!  We can provide you with guidance and code review in return.  If so, tick any of the relevant boxes below:

- [ ] I'd like to try adding this scraper myself
- [ ] I'd like guidance to help me develop a scraper
- [x] I'd prefer if the `recipe-scrapers` community try to add this

-------

Some additional hints:

- There are different languages for the website available, e.g. https://www.kitchenstories.com/en/ or https://www.kitchenstories.com/de/
- It sometimes works already, but not for everything (e.g. servings are always 0)

### PR DESCRIPTION
**Pull request by astappiev**
Not sure why do we need a scraper for a source that perfectly fits Schema.org/Recipe, but I'm using the website as a source too and saw a request to add a scraper.

closes #547

### CODE REVIEW

File: recipe_scrapers/kitchenstories.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        return self.schema.total_time()
+
+    def cook_time(self):
+        return self.schema.cook_time()
+
+    def prep_time(self):
+        return self.schema.prep_time()
+
+    def yields(self):
+        return self.schema.yields()
```
Comment (jayaddison) [inline recipe_scrapers/kitchenstories.py:25]: Similarly perhaps we should omit the `yields` implementation here.

File: tests/kitchenstories.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+from recipe_scrapers.kitchenstories import KitchenStories
+from tests import ScraperTest
+
+
+class TestKitchenStoriesScraper(ScraperTest):
```
Comment (jayaddison) [inline tests/kitchenstories.py:5]: A request to add test coverage on the `language` method given the multilingual site content.

Comment (astappiev) [reply tests/kitchenstories.py:5]: Should I put multiple html files and add multiple tests for each language?

File: recipe_scrapers/kitchenstories.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def cook_time(self):
+        return self.schema.cook_time()
```
[the remark this answers is not in the document]
Comment (jayaddison) [reply recipe_scrapers/kitchenstories.py:19]: Would you feel like writing code to infer times?  If so, we could do that in a separate pull request, and then merge it before this one.

No problem if not: I think the code is generally fine as-is, and if there are cases where all three values are present, then the code is valid.

Some of the handling around `None` and zero values in `get_minutes` looks a little confusing at the moment, too.  Possibly another issue to file.

[the remark this answers is not in the document]
Comment (astappiev) [reply recipe_scrapers/kitchenstories.py:19]: I keep it as it is, should be discussed separately

[the remark this answers is not in the document]
Comment (jayaddison) [reply recipe_scrapers/kitchenstories.py:19]: Agreed, we have some technical debt there to clean up :/  Thanks for looking into it.  I'll open an issue for that soon too (although might ask for your help filling in the details, while you're familiar with it)

[the remark this answers is not in the document]
Comment (jayaddison) [reply recipe_scrapers/kitchenstories.py:19]: Filed a ticket for these improvements at #551 - when you have time, please have a read and add any more information you like.

**Review by jayaddison — APPROVED** [review]

**Pull request merged**